cloudinstall/charms/ceph.py

```diff
@@ -29,7 +29,8 @@ class CharmCeph(CharmBase):
     display_name = 'Ceph'
     menuable = True
     display_priority = DisplayPriorities.Storage
-    related = ['glance', 'nova-compute']
+    related = {'glance': ('glance:ceph', 'ceph:client'),
+               'nova-compute': ('nova-compute:ceph', 'ceph:client')}
```

Comment:
actually.. no, not that i can see.. hmmm need to look over it again to see if I had a good reason if not ill fix it